### PR TITLE
Extend base realsense rs_launch.py file

### PR DESCRIPTION
**Pull request by dyackzan**
This allows us to make use of the recommended realsense camera defaults and makes us more robust to realsense updates in the future. This proper parameterization adds color data back into the depth cloud again, resolving https://github.com/PickNikRobotics/moveit_studio/issues/8189

### CODE REVIEW

File: src/picknik_ur_base_config/launch/pro_rs_launch.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+from launch.actions import OpaqueFunction
+
+# Add the realsense2_camera/launch directory to the user's path so we can import the realsense2_camera rs_launch module.
+realsense2_camera_path = get_package_share_directory("realsense2_camera")
+sys.path.append(f"{realsense2_camera_path}/launch")
+
+import rs_launch
+
+
+rs_launch.configurable_parameters.extend(
```
Comment (pac48) [inline src/picknik_ur_base_config/launch/pro_rs_launch.py:14]: Can we move this to the `generate_launch_description`?

File: src/picknik_ur_base_config/launch/pro_rs_launch.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+from ament_index_python import get_package_share_directory
+from launch import LaunchDescription
+from launch.actions import OpaqueFunction
+
+# Add the realsense2_camera/launch directory to the user's path so we can import the realsense2_camera rs_launch module.
+realsense2_camera_path = get_package_share_directory("realsense2_camera")
+sys.path.append(f"{realsense2_camera_path}/launch")
+
+import rs_launch
```
Comment (pac48) [inline src/picknik_ur_base_config/launch/pro_rs_launch.py:11]: What about import rs_launch.configurable_parameters as configurable_parameters?

**Review by EzraBrooks — COMMENTED** [review]
Does this require changes to the camera config migration script?

**Comment by dyackzan** [comment]
> Does this require changes to the camera config migration script?

No, the camera launch file in the user configs is unchanged

**Review by EzraBrooks — APPROVED** [review]
I think I'm okay with this since it doesn't change the user-space launch files, even though I'm not sure I like the complexity of importing another launch file and mucking around in it.

**Pull request merged**